ledger/error-definitions/error-definitions-lf/BUILD.bazel

```diff
@@ -0,0 +1,25 @@
+load(
+    "//bazel_tools:scala.bzl",
+    "da_scala_library",
+)
+
+da_scala_library(
+    name = "error-definitions-lf",
```

Comment:
Is it intentional that this Bazel package contains nothing but the `BUILD.bazel`?